index.html

```diff
@@ -35,7 +35,119 @@ <h1>I'm David, Glad to see you</h1>
                 <li><a href="#"><img src="./img/hand.png" alt="hand icon"></a></li>
             </ul>
         </div>
-        
+    </section>
+    <section class="cards-container">
+        <div class="card">
+            <img class="card-image" src="./img/portfolio1.jpg" alt="webpage image">
+            <h2>Tonic</h2>
+            <div class="tags">
+                <p>CANOPY</p>
+                <img class="bullet" src="./img/Counter.png" alt="bullet">
+                <span>Back End Dev</span>
+                <img src="./img/Counter.png" alt="bullet">
+                <span>2015</span>
+            </div>
+            <p class="description">
+                A daily selection of privately personalized reads. No accounts or 
+                sign-ups required.
+            </p>
+            <ul class="technologies">
+                <li>html</li>
+                <li>css</li>
+                <li>javaScript</li>
+            </ul>
+            <div>
+                <button type="button">See Project</button>
+            </div>
+            
+        </div>
+        <div class="card">
+            <img class="card-image" src="./img/portfolio2.jpg" alt="webpage image">
+            <h2>Tonic</h2>
+            <div class="tags">
+                <p>CANOPY</p>
+                <img src="./img/Counter.png" alt="bullet">
+                <span>Back End Dev</span>
+                <img src="./img/Counter.png" alt="bullet">
+                <span>2015</span>
```

Comment:
- Kindly those bullets are large compared to the Figma design,  please minimize its size to stick to the Figma design 👌👍


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/212357456-2228430d-563e-4e16-9b1d-d60b0c313569.png" width="200">|<img src="https://user-images.githubusercontent.com/52010789/212357463-7cbd93bb-2550-411a-9ec8-4e792181d23b.png" width="200">|


😎️

🤝️